### ISSUE TITLE
Move books to a separate page

### ISSUE THREAD

**Issue by satlive** [satlive/website#4]
The books about SAT are currently displayed on the side. I am thinking about moving them on a separate web page.

Feedback about this idea is welcome.

**Comment by Mystelven** [comment]
I agree with this idea, for now, except being hover on the book, we don't have any informations about the books. 

By Putting them on a separate web page, we will allow more freedom to add informations about them :)
